src/config.rs

```diff
@@ -33,8 +32,7 @@ const NEW_LINE: &str = "\n";
 
 /// Initializes env_logger and prints metadata
 pub fn init() {
-    set_var(RUST_LOG_KEY, LOG_LEVEL);
```

Comment:
For the reason behind this change, the `set_var` function is marked as unsafe in the 2024 edition of Rust -- read more [here](https://doc.rust-lang.org/edition-guide/rust-2024/newly-unsafe-functions.html). Since `gpm` is single-threaded, it should still be fine to use it, but the `env_logger` has a built-in way to set the log level so I just used that.